src/main/java/Mikey.java

```diff
@@ -0,0 +1,295 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.StringJoiner;
+import java.util.Arrays;
+
+public class Mikey {
+
+    public static ArrayList<Task> tasks = new ArrayList<Task>();
+
+    public static Task newTodo(String taskName, int completion) {
```

Comment:
The name of this method does not exactly describe what the method does. Consider naming it `createNewToDo` or `makeToDo` or something along those lines. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions)